docs/technical-reference/tasks.md

```diff
@@ -67,7 +65,7 @@ You can personalize the way in which your SelfInstructTask behaves by changing t
 Let's see an example of how to customise a SelfInstructTask to create Haikus in the snippet below. You can take a look at this dataset as an example of a [Haiku DPO dataset](https://huggingface.co/datasets/davanstrien/haiku_dpo).
 
 ```python
---8<-- "docs/snippets/technical-reference/tasks/custom_task_selfinstruct_haikus.py"
+--8<-- "docs/snippets/technical-reference/tasks/custom_task_selfinstruct_haikus_docs.py"
```

Comment:
@I think it might be cleaner to add this one to the rest of the examples?

Reply:
with this, I mean the haiku example

Reply:
It was just a name change, I can revert it, but thought it would be useful to state that this snippet is for the docs

Reply:
so, we leave the SelfInstructTask with no example and we add the Haiku example to all the example of CustomTasks?